ProteoformSuiteGUI/AggregatedProteoforms.cs

```diff
@@ -82,13 +141,5 @@ private void nUD_Missed_Ks_ValueChanged(object sender, EventArgs e)
             Lollipop.aggregate_proteoforms();
         }
 
-        private void dgv_AcceptNeuCdLtProteoforms_CellContentClick(object sender, DataGridViewCellEventArgs e)
```

Comment:
I deleted this shortly after I had made the commit when I added it because it's unfinished/ wasn't doing anything - I had added it a while ago to try to get the accept/not accept boolean working but then stopped half way to work on other stuff... I figured we could add it back then.